packages/forms-web-app/__tests__/unit/utils/is-before-or-after-date.test.js

```diff
@@ -51,6 +52,27 @@ describe('#utils/is-before-or-after-date', () => {
 		});
 	});
 
+	describe('#isBeforeTodayUTC', () => {
+		describe('When given datetime is today 09:00:00', () => {
+			describe('and the current datetime is today 12:00:00', () => {
+				it('should return false', () => {
+					jest.useFakeTimers().setSystemTime(new Date('2020-01-01 12:00:00'));
+					const result = isBeforeTodayUTC('2020-01-01 09:00:00');
+					expect(result).toEqual(false);
+				});
+			});
+		});
+		// today 00:00:00 should be treated as 23:59:59
+		describe('When given datetime is today 00:00:00', () => {
+			describe('and the current datetime is today 12:00:00', () => {
+				it('should return true', () => {
+					jest.useFakeTimers().setSystemTime(new Date('2020-01-01 12:00:00'));
+					const result = isBeforeTodayUTC('2020-01-01 00:00:00');
+					expect(result).toEqual(true);
+				});
+			});
+		});
+	});
```

Comment:
maybe add a test case for a string with no time?

Reply:
Ah I did actually have that but removed it because the test cases was just duplicating the tests we have for the `isBeforeNowUTC` function. But I can add it if you think its worth having here too?

Reply:
Added on commit: https://github.com/Planning-Inspectorate/applications-service/pull/786/commits/2df44c0dfa13a4b22110838d8a41bf336b8d8cad

Reply:
yeah worth having i think, as adding a time where it is missing is the main thing ` isBeforeTodayUTC` is doing

Reply:
https://github.com/Planning-Inspectorate/applications-service/pull/786/commits/dfa0f356a7d3e7d579aae63aea8b952ea892a5b7